models/Subscriber.js

```diff
@@ -1,9 +1,9 @@
 var mongoose = require('mongoose');
-var twilio = require('twilio');
+const Twilio = require('twilio');
 var config = require('../config');
 
 // create an authenticated Twilio REST API client
-var client = twilio(config.accountSid, config.authToken);
+const client = new Twilio(config.accountSid, config.authToken);
```

Comment:
There is no need to use the new operator over here. As you can see the [`initializer`](https://github.com/twilio/twilio-node/blob/9e1e45b5fd2f95e30dd243f3ca483d07f0d20c07/lib/index.js#L6) function you're using under the hood will do it for you.

In fact, I'll use the `twilio` lowercased since we're referring to the library. Besides that for this case I use `var` instead of `const` to keep an homogeneous style in this module.

What do you think?